lib/settingslogic.rb

```diff
@@ -95,16 +96,16 @@ def initialize(hash_or_file = self.class.source, section = nil)
     #puts "new! #{hash_or_file}"
     case hash_or_file
     when nil
-      raise Errno::ENOENT, "No file specified as Settingslogic source"
+      Kernel.raise Errno::ENOENT, "No file specified as Settingslogic source"
```

Comment:
These Kernel prefix is needed because `SimpleDelegator` is a `BasicObject`, they don't have kernel included.